pylintrc_tweaks

```diff
@@ -2,10 +2,13 @@
 [MASTER]
 ignore+ = ,.git,.tox,migrations,node_modules,.pycharm_helpers
 init-hook="import sys; sys.path.extend(['lms/djangoapps', 'cms/djangoapps', 'common/djangoapps'])"
+load-plugins+=,caniusepython3.pylint_checker
```

Comment:
what is caniusepython3.pylint_checker?

Reply:
It checks for a couple of incompatibilities in std library imports, I think (along with u-prefix strings, but too aggressively).